### ISSUE TITLE
Improve mouse-over detection of tree view expand/collapse "rotating triangles"

### ISSUE THREAD

**Issue by FunctionPoint** [dolphinsmalltalk/Dolphin#599]
In any tree view, for example a class view, there are "rotating triangles" to expand or collapse a branch. When a branch is to be expanded, it looks like a ">", and before collapsing, it looks like a "v", just like in the Windows Explorer.
.
Now I would often click on an expand-collapse triangle and nothing happened a few times. After some checking I see that it's caused by a skewed mouse detection on the traingle. 
.
If you approach a rotating traingle from the *left* side with your mouse pointer, it will only light-up (for expand or collapse) if the mouse is already way past the left side of the triangle image. So clicking 1 pixel left of that point will not highlight it, or expand/collapse it when clicking.
.
If you approach a rotating traingle from the *right* side with your mouse pointer, it will already light up when the mosue is over the dotted line on the right side of the triangle. So not even on the triangle image yet.
.
The photo below shows a rotating triangle that is *not* lit up in light-blue, even though the mouse pointer is already over the image of it. 
Windows Explorer will show the desired detection behaviour.

**Comment by FunctionPoint** [comment]
![mouseover](https://user-images.githubusercontent.com/9684694/44551786-7055c600-a728-11e8-994d-50e8d06c3139.jpg)

**Comment by FunctionPoint** [comment]
Aha,
I had looked at it for a short while,
but couldn't find any hooks in Dolphin too.
Hoped you might know one .
.
Well, it's not a showstopper...

**Comment by FunctionPoint** [comment]
A workaround seems to be to set the indent to 16 pixels.
Then there is no room for the horizontal line that offsets the horizontal detection wrongly.

**Comment by blairmcg** [comment]
Yes, I'd played with that (which is how I realised it was related to that setting). I don't believe it completely solves the issue; the hot zone is still offset, just not enough to really notice. I think the best thing to do is to allow the property to be defaulted (i.e. be nil), and then interpret this as allowing the control to uses its default indent setting. The default is actually 19, at least on Windows 10 with 16x16 icons. I've tried this and it seems a reasonable compromise between insufficient indent, which makes it look rather cramped (and in `noIcons` mode, causes the tumbler to shrink) and a larger indent which then starts to make the bug annoying.
